### ISSUE TITLE
head css files sort order

### ISSUE THREAD

**Issue by Baworder** [magento/magento2#8406]
### Preconditions

1. Magento 2.1.2

### Steps to reproduce

I am developing a Magento 2 shop based on the Bootstrap library.
But the Bootstrap library should only get included in cms pages. So I added the following code in the file `Magento_Cms/layout/cms_page_view.xml`:
`<head>`
`<css src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" src_type="url" />`
`</head>`

The problem is, that the bootstrap CSS file gets included after the `styles-l.css` and `styles-m.css` files (shoud get included before).
So I did a bit of research in the web. The often recomended way to order files with the `order="x"` tag does not work.

Is there an other way to do that?

**Comment by Igloczek** [comment]
Use `<css src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" src_type="url" order="10" />`. By defult CSS elements have order set to `0`, so adjusting this value you can create proper loading order.

BTW. It's not an issue report and you should close it.
If you are looking for help ask question on [Magento Community Forum](http://community.magento.com/) or on [StackExchange](http://magento.stackexchange.com/)

**Comment by Baworder** [comment]
@Igloczek Thank you for your answer!
But that's not working for me.
```
<link  rel="stylesheet" type="text/css"  media="all" href="http://domain.tld/static/_cache/merged/f20819a53f99bb909d9eb94305d9b4e9.css" />
<link  rel="stylesheet" type="text/css"  order="3" after="-" href="http://domain.tld/static/frontend/[A]/[B]/de_DE/css/styles-m.css" />
<link  rel="stylesheet" type="text/css"  media="screen and (min-width: 768px)" order="4" href="http://domain.tld/static/frontend/[A]/[B]/de_DE/css/styles-l.css" />
<link  rel="stylesheet" type="text/css"  media="print" href="http://domain.tld/static/frontend/[A]/[B]/de_DE/css/print.css" />
<link  rel="stylesheet" type="text/css"  order="1" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
<link  rel="stylesheet" type="text/css"  order="2" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" />
```

I will ask for help at StackExchange.

**Comment by kirill533** [comment]
Workaround:

Rename bootstrap.css into bootstrap.less.
Includ bootstrap.less as a regular less file as part of the theme at the start of the _sources.less file.
